microsoft-edge/visual-studio/index.md

```diff
@@ -34,52 +34,49 @@ The simplified workflow allows you to debug JavaScript that run in Microsoft Edg
 
 ### Create a new ASP.NET Core web app  
 
-Open Visual Studio 2019 and choose **Create a new project**.  On the next screen, choose **ASP.NET Core Web app** > **Next**.  
+Open Visual Studio 2019 and select **Create a new project**.  In the search box on the next screen, enter **react**.  Select **ASP.NET Core with React.js** from the list of templates and then **Next**.  
 
-:::image type="complex" source="./media/create-new-project.png" alt-text="Create a new ASP.NET Core Web app" lightbox="./media/create-new-project.png":::
-   Create a new ASP.NET Core Web app  
-:::image-end:::  
+This **React.js** template specifies how to integrate React.js with an ASP.NET Core app.  
 
-Provide a **Project name** for your new project and choose **Create**.  For the purposes of the example, choose **React.js** as the template, and choose **Create**.  The **React.js** template specifies how to integrate React.js with an ASP.NET Core app.  
+:::image type="complex" source="./media/create-new-project.png" alt-text="Create a new ASP.NET Core Web app with React.js" lightbox="./media/create-new-project.png":::
```

Comment:
If not too much trouble/time, in screenshots, change yellow box to red.  Usually 3px not 5px, but that width could be hard to change.  
To change yellow to red in Paint:
1. Shift+click an image link in the .md file in VS Code.
2. Use a menu item or Sh+Alt+R to open Explorer at the dir.
3. Right-click an image file, Edit, to open in Paint.
4. Select or enter a red color.  Or open an image with red box and use Dropper tool to sample it.
4. Select Paint Bucket tool, click a yellow rectangle.  Save.